payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetPaymentController.kt

```diff
@@ -0,0 +1,54 @@
+package com.stripe.android.paymentsheet
+
+import androidx.activity.result.ActivityResultCallback
+import androidx.activity.result.ActivityResultCaller
+import com.stripe.android.StripeIntentResult
+import com.stripe.android.model.ConfirmStripeIntentParams
+import com.stripe.android.model.StripeIntent
+import com.stripe.android.payments.PaymentFlowResult
+import com.stripe.android.payments.PaymentFlowResultProcessor
+import com.stripe.android.paymentsheet.model.StripeIntentValidator
+import com.stripe.android.view.AuthActivityStarterHost
+import kotlin.coroutines.CoroutineContext
+
+sealed interface StartAndConfirmResult {
```

Comment:
```suggestion
/**
 * This sealed interface is the result of startAndConfirm it can either be an Error, Fatal, success, or a StripeIntent that is still ready for a new selection
 */
sealed interface StartAndConfirmResult {
```

Reply:
ah, so maybe we have a head start on it then!